modules/testcontainers/src/main/kotlin/jp/co/soramitsu/iroha2/testcontainers/IrohaContainer.kt

```diff
@@ -91,7 +91,7 @@ open class IrohaContainer : GenericContainer<IrohaContainer> {
 
     companion object {
         const val NETWORK_ALIAS = "iroha"
-        const val DEFAULT_IMAGE_TAG = "stable"
+        const val DEFAULT_IMAGE_TAG = "dev"
```

Comment:
don't do this please, specify a build hash instead (dev-nightly or smth or wait for a tag from the Iroha team)